tests/yast2_gui/yast2_firewall_set_service_port.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: YaST2 Firewall UI test checks verious configurations and settings of firewall
+# Make sure yast2 firewall can opened properly. Configurations can be changed and written correctly.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use network_utils 'iface';
+use YaST::Module;
+use version_utils "is_sle";
+
+sub verify_interface {
+    my (%args) = @_;
+    validate_script_output("firewall-cmd --list-interfaces --zone=$args{zone}", sub { m/$args{device}/ }, proceed_on_failure => 1);
+}
+
+sub configure_zone {
+    my (%args) = @_;
+    $testapi::distri->get_firewall()->select_zone_page($args{zone});
+    save_screenshot;
+    $testapi::distri->get_firewall()->add_service($args{zone}, $args{service});
+    save_screenshot;
+    $testapi::distri->get_firewall()->add_tcp_port($args{port});
+    save_screenshot;
+}
+
+sub run {
+    my $self = shift;
+    my $iface = iface();
+
+    record_info('Zones', "Configure zone adding service and port");
```

Comment:
let's remove all those `record_info` with this atomicity are not needed, we can just refer to the description of the test.